plugins/jira-plugin-ideas/README.md

```diff
@@ -11,3 +11,233 @@ solution_tags:
 systems:
 - jira
 ---
+
+# Introduction
+
+Jira is a leading platform for managing projects and ideas across industries. Jira Backlog Ideas simplifies retrieving ideas or tickets from your backlog by filtering based on project names and keywords, helping teams prioritize and act efficiently.
+
+Integrating this feature with your bot allows you to fetch and manage backlog ideas instantly through conversational queries, boosting productivity.
+
+This guide walks you through adding the "Check Backlog Ideas" feature to your bot using Creator Studio.
+
+Let's get started!
+
+# Prerequisites
+
+- [Postman](https://www.postman.com/) or an API Testing Tool
+- Jira Connector built in Creator Studio (follow the [Jira Authentication guide](https://developer.moveworks.com/creator-studio/resources/connector/?id=jira) to create your connector)
+
+# **What are we building?**
+
+## **Conversation Design**
+
+This [purple chat](https://developer.moveworks.com/creator-studio/developer-tools/purple-chat-builder/?workspace=%7B%22title%22%3A%22My+Workspace%22%2C%22botSettings%22%3A%7B%7D%2C%22mocks%22%3A%5B%7B%22id%22%3A6991%2C%22title%22%3A%22Mock+1%22%2C%22transcript%22%3A%7B%22settings%22%3A%7B%22colorStyle%22%3A%22LIGHT%22%2C%22startTime%22%3A%2211%3A43+AM%22%2C%22defaultPerson%22%3A%22GWEN%22%2C%22editable%22%3Atrue%7D%2C%22messages%22%3A%5B%7B%22from%22%3A%22USER%22%2C%22text%22%3A%22%3Cp%3EAre+there+any+plugin+ideas+about+approvals%3F%3Cbr%3E%3C%2Fp%3E%22%7D%2C%7B%22from%22%3A%22ANNOTATION%22%2C%22text%22%3A%22%3Cp%3E%E2%8F%B3+Calling+Plugin+%3Cb%3ECheck+Plugin+Ideas+%3C%2Fb%3Efor+%5C%22approvals%5C%22%3Cbr%3E%3C%2Fp%3E%22%7D%2C%7B%22from%22%3A%22BOT%22%2C%22text%22%3A%22%3Cp%3ESure%2C+here+are+the+currently+open+plugin+ideas.%3Cbr%3E%3C%2Fp%3E%22%2C%22cards%22%3A%5B%7B%22title%22%3A%22%3Cp%3EPlugin+Idea%3A+Purchase+Order+Approvals+from+Coupa%3Cbr%3E%3C%2Fp%3E%22%2C%22buttons%22%3A%5B%7B%22style%22%3A%22PRIMARY%22%2C%22text%22%3A%22View+Details%22%7D%2C%7B%22text%22%3A%22Upvote%22%7D%5D%7D%2C%7B%22title%22%3A%22%3Cp%3EPlugin+Idea%3A+Expense+Report+Approvals+from+Expensify%3Cbr%3E%3C%2Fp%3E%22%2C%22buttons%22%3A%5B%7B%22style%22%3A%22PRIMARY%22%2C%22text%22%3A%22View+Details%22%7D%2C%7B%22text%22%3A%22Upvote%22%7D%5D%7D%5D%7D%5D%7D%7D%5D%7D) shows the experience we are going to build.
```

Comment:
Let's update to purple chat to: https://developer.moveworks.com/creator-studio/developer-tools/purple-chat-builder/?workspace=%7B%22title%22%3A%22My+Workspace%22%2C%22botSettings%22%3A%7B%22name%22%3A%22%22%2C%22imageUrl%22%3A%22%22%7D%2C%22mocks%22%3A%5B%7B%22id%22%3A9926%2C%22title%22%3A%22New+Mock%22%2C%22transcript%22%3A%7B%22messages%22%3A%5B%7B%22from%22%3A%22USER%22%2C%22text%22%3A%22Show+me+my+Jira+backlog+items.%22%7D%2C%7B%22from%22%3A%22ANNOTATION%22%2C%22text%22%3A%22Retrieves+user%27s+backlog+items+from+Jira.%22%7D%2C%7B%22from%22%3A%22BOT%22%2C%22text%22%3A%22Here%27s+your+current+Jira+backlog%3A%22%2C%22cards%22%3A%5B%7B%22title%22%3A%22Backlog+Item+1%3A+Implement+login+feature%22%2C%22text%22%3A%22%3Cb%3EStatus%3A%3C%2Fb%3E+To+Do%3Cbr%3E%3Cb%3EPriority%3A%3C%2Fb%3E+High%22%7D%2C%7B%22title%22%3A%22Backlog+Item+2%3A+Fix+email+notification+bug%22%2C%22text%22%3A%22%3Cb%3EStatus%3A%3C%2Fb%3E+To+Do%3Cbr%3E%3Cb%3EPriority%3A%3C%2Fb%3E+Medium%22%7D%2C%7B%22title%22%3A%22Backlog+Item+3%3A+Update+user+profile+page%22%2C%22text%22%3A%22%3Cb%3EStatus%3A%3C%2Fb%3E+To+Do%3Cbr%3E%3Cb%3EPriority%3A%3C%2Fb%3E+Low%22%7D%5D%7D%5D%2C%22settings%22%3A%7B%22colorStyle%22%3A%22LIGHT%22%2C%22startTime%22%3A%2211%3A43%2BAM%22%2C%22defaultPerson%22%3A%22GWEN%22%2C%22editable%22%3Atrue%2C%22botName%22%3A%22%22%2C%22botImageUrl%22%3A%22%22%7D%7D%7D%5D%7D